### DISCUSSION TITLE
What are the advantages of non-alphanumeric strips?

### DISCUSSION THREAD

**Discussion by kotaroyamazaki** [aarondl/sqlboiler#1124] [Q&A] [answered]
Non-alphabetic characters in Enum have been omitted in v 4.10, what is the advantages of this modification?
https://github.com/volatiletech/sqlboiler/commit/15560e61db2bafa68317fedd7dc02a1aaa26a713

In our project, compatibility was destroyed and ENUM constants, which was named in the upper snake case, became significantly less readable.

**Comment by stephenafamo** [comment, marked as answer]
This was a mistake. It was initially to make sure that only valid go identifiers remain and somehow, we forgot about underscores. It should be reverted in the next release. My apologies.

**Reply by stephenafamo** [reply]
Fixed in `v4.11.0`
